File: Chris/2022-06-21-NumberOf1Bits.cs
public class Solution {
    public int HammingWeight(uint n) {
        int numOnes=0;
        uint currDigit;
        while(n != 0){
            currDigit = n & 1;
            if(currDigit==1){
                numOnes++;
            }
            n = n >> 1;
        }
        return numOnes;
    }
}

File: MinjiJu/2022-06-21-number-of-1-bits.cpp
class Solution {
public:
    int hammingWeight(uint32_t n) {
        
        int count = 0;
        
        while(n>0){
            count += n & 1; // increment counter if 1 found
            n >>= 1;        // bitshift n
        }
        
        return count;
    }
};